packages/protocol/test-sol/devchain/e2e/utils.sol

```diff
@@ -1,52 +1,103 @@
-
-// All core contracts that are expected to be in the Registry on the devchain
-import "@celo-contracts-8/common/FeeCurrencyDirectory.sol";
-import "@celo-contracts/stability/interfaces/ISortedOracles.sol";
-import "@celo-contracts/common/interfaces/ICeloUnreleasedTreasury.sol";
-
-import "@test-sol/TestWithUtils08.sol";
 
 contract Devchain is TestWithUtils08 {
   // All core contracts that are expected to be in the Registry on the devchain
-  ISortedOracles sortedOracles;
-  FeeCurrencyDirectory feeCurrencyDirectory;
-  IEpochManager epochManagerContract;
-  ICeloUnreleasedTreasury celoUnreleasedTreasuryContract;
-  IValidators validators;
+  // TODO: Change all contracts to be imported as interfaces
   IAccounts accounts;
-  IScoreManager scoreManager;
-  IElection election;
-  ILockedCelo lockedCelo;
+  // BlockchainParameters blockchainParameters;
+  ICeloUnreleasedTreasury celoUnreleasedTreasuryContract;
   ICeloToken celoTokenContract;
+  // DoubleSigningSlasher doubleSigningSlasher;
+  // DowntimeSlasher downtimeSlasher;
+  IElection election;
+  IEpochRewards epochRewards;
+  IEpochManagerEnabler epochManagerEnablerContract;
+  IEpochManager epochManagerContract;
+  IEscrow escrow;
+  IFederatedAttestations federatedAttestations;
+  // IFeeCurrencyWhitelist feeCurrencyWhitelist;
+  FeeCurrencyDirectory feeCurrencyDirectory;
+  IFeeHandler feeHandler;
+  IFreezer freezer;
+  IGovernance governance;
+  // GovernanceSlasher governanceSlasher;
+  ILockedGold lockedCelo;
+  IOdisPayments odisPayments;
+  IRandom randomContract;
+  IScoreManager scoreManager;
+  ISortedOracles sortedOracles;
+  IValidators validators;
 
   constructor() {
     // Fetch all core contracts that are expeceted to be in the Registry on the devchain
-    sortedOracles = getSortedOracles();
-    feeCurrencyDirectory = FeeCurrencyDirectory(
-      registryContract.getAddressForStringOrDie("FeeCurrencyDirectory")
-    ); // FeeCurrencyDirectory is not in UsingRegistry.sol
-
-    epochManagerContract = getEpochManager();
-    celoUnreleasedTreasuryContract = getCeloUnreleasedTreasury();
-    validators = getValidators();
+    // TODO: Ensure all contracts have getters in UsingRegistry
     accounts = getAccounts();
-    scoreManager = IScoreManager(address(getScoreReader()));
-    election = getElection();
-    lockedCelo = getLockedCelo();
+    // blockchainParameters = BlockchainParameters(
+    //   registryContract.getAddressForOrDie(BLOCKCHAIN_PARAMETERS_REGISTRY_ID)
+    // ); // FIXME: BlockchainParameters is not in UsingRegistry.sol
+    celoUnreleasedTreasuryContract = getCeloUnreleasedTreasury();
     celoTokenContract = ICeloToken(registryContract.getAddressForOrDie(GOLD_TOKEN_REGISTRY_ID));
+    // doubleSigningSlasher = DoubleSigningSlasher(
+    //   registryContract.getAddressForOrDie(DOUBLE_SIGNING_SLASHER_REGISTRY_ID)
+    // ); // FIXME: DoubleSigningSlasher is not in UsingRegistry.sol
```

Comment:
<img width="716" alt="Screenshot 2025-05-20 at 14 26 49" src="https://github.com/user-attachments/assets/0588ba17-ca76-4d6e-b817-24993b3ebcaa" />

Reply:
see comment above as to why they are no longer in registry.